bin/currency/update-prod-dependencies.js

```diff
@@ -0,0 +1,104 @@
+/*
+ * (c) Copyright IBM Corp. 2025
+ */
+
+'use strict';
+
+const path = require('path');
+const { execSync } = require('child_process');
+const utils = require('./utils');
+const BRANCH = process.env.BRANCH;
+const SKIP_PUSH = process.env.SKIP_PUSH === 'true';
+const MAX_PROD_PR_LIMIT = process.env.MAX_PROD_PR_LIMIT || 5;
+const cwd = path.join(__dirname, '..', '..');
+
+if (!BRANCH) throw new Error('Please set env variable "BRANCH".');
+
+console.log(`BRANCH: ${BRANCH}`);
```

Comment:
Putting it here:

IMO both procedures (currency updates + prod updates) run through the same steps:

1. source of dependencies (currencies.json vs prod deps in package.json)
2. update phase (IMO is equal -> git operations, slighlty different config e.g. PR limit or minor vs major handling)

It would be good to extract more common logic into the utils.js.